### DISCUSSION TITLE
glAccountId miért maxOccurs 2 a VatDeclarationData/ VatAnalitcs/ VatAnaliticsItem-ben?

### DISCUSSION THREAD

**Discussion by szalaierika** [nav-gov-hu/eVAT#36] [Kérdések és válaszok - Gépi interfész] [answered]
Kedves NAV Szakértők!

- A glAccountId: miért maxOccurs 2  a VatDeclarationData/ VarAnalitcs/ VatAnaliticsItem-ben?
- Milyen főkönyvi számlák ezek?
- ÁFA-főkönyvi számla?

Ha ÁFA-főkönyvi számla, akkor az a taxInformation node-ba lenne való. Illetve akkor is több lehet, mint kettő, mert a 466-t és 467-t sokfelé bonthatja az adózó.

Köszönjük előre is a válaszokat!

**Comment by NTCA-HM** [comment]
Kedves @szalaierika !

1.)	Az áfaanalitika opcionálisan tartalmazhat főkönyvi adatokat a generalLedgerInformation csomópontban. A főkönyvi adatok szerepeltetése XSD alapján nem kötelező. A főkönyvi számla számát a főkönyvi nyilvántartásban szereplő bontásban kell megadni. Tehát ha a 466-os és 467-es áfa főkönyvi számlát több felé bontja az adózó, akkor valóban lehet 2-nél több is, de tranzakciónként maximum kettő szerepelhet összességében (Levonható/Fizetendő). Ezért van az előfordulás korlátozva.
2.)	Igen Áfa főkönyvi számla
3.)	Lásd az 1.)-es választ

**Reply by szalaierika** [reply]
Ahogy írtuk korábban, a 466-t és 467-t többfelé bonthatja az adózó, így egy számlában több, mint 2 áfa-számla tud szerepelni egyidejűleg. Ebben az esetben melyik 2 számlát szerepeltessük az adatszolgáltatásban?

Pl. Kimenő számla 3 különböző adókóddal és 3 különböző 467 számlával (467100, 467200, 467300). Mi az elvárás ilyen esetben?

**Reply by szalaierika** [reply]
Kedves @NTCA-HM !

Azt elfelejtettem kihangsúlyozni, hogy meglátásunk szerint a generalLedgerInformation node a taxInformation node alatt kellene, hogy legyen, mivel áfa-tétel szinten logikus a főkönyvi számlaszám tárolása, nem pedig bizonylatszinten.

Köszönjük!

**Reply by renced42** [reply, marked as answer]
Kedves @szalaierika 

Az analitikában egy tételsorban („line” -ban)  több standard adókód is szerepeltethető, így ezzel az olyan számlákat lehet kezelni, ahol többféle standardadókódot is alkalmazni kell (pl.: egy számla, amely többféle fizetendő adókulcsos tételt tartalmaz). Ugyanakkor főkönyvi számla adat VatAnalyticsItem-enként csak egy lehet.
Egy számla esetében tehát az azonos standard adókód alá tartozó számlatételeket aggregálva a megfelelő standard adókód alatt kell szerepeltetni az analitikában, függetlenül attól, hogy azok a számlatételek esetleg a könyvelésben különböző főkönyvi számlákon vannak alábontva (pl. 467100, 467200, 467300).
 
Ugyanakkor, ahogyan az a korábbi válaszban is szerepelt, az áfaanalitika opcionálisan tartalmazhat főkönyvi adatokat a generalLedgerInformation csomópontban, így azok megadása nem kötelező.
